diff --git a/node_modules/react-native-material-initials/native.js b/node_modules/react-native-material-initials/native.js
index e54f99d..a0f9971 100644
--- a/node_modules/react-native-material-initials/native.js
+++ b/node_modules/react-native-material-initials/native.js
@@ -62,4 +62,5 @@ const styles = StyleSheet.create({
   }
 });
 
-module.exports = NativeIniticon;
+//module.exports = NativeIniticon;
+export default NativeIniticon;
